sdk/servicebus/Azure.Messaging.ServiceBus/tests/Administration/ServiceBusManagementClientLiveTests.cs

```diff
@@ -207,20 +232,18 @@ public async Task BasicTopicCrudOperations()
 
             TopicProperties createdTopic = createdTopicResponse.Value;
 
-            if (Mode == RecordedTestMode.Playback)
+            if (premium)
             {
-                // Auth rules use a randomly generated key, but we don't want to store
-                // these in our test recordings, so we skip the auth rule comparison
-                // when in playback mode.
-                Assert.AreEqual(options, new CreateTopicOptions(createdTopic) { AuthorizationRules = options.AuthorizationRules.Clone() });
-                Assert.AreEqual(createdTopic, new TopicProperties(options) { AuthorizationRules = createdTopic.AuthorizationRules.Clone() });
+                Assert.AreEqual(100000, createdTopic.MaxMessageSizeInKilobytes);
             }
             else
             {
-                Assert.AreEqual(options, new CreateTopicOptions(createdTopic));
-                Assert.AreEqual(createdTopic, new TopicProperties(options));
+                // standard namespaces use 256KB
```

Comment:
This seems a bit dangerous.   What if the service changes the limit?   Should we just assert that it's less than our requested amount?   Still brittle, but a bit less so...

Reply:
Worst case scenario, the test breaks if the service increases the limit and then we fix the test.

Reply:
Maybe we could leave more details in the comment, just in case...    "hey, the service controls this value, if it fails check to see if the service limits have increased?"